Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -965,23 +965,6 @@ public function __construct()
         $this->makeTest($expected);
     }
 
-    public function testStatic()
-    {
-        $expected = <<<'EOF'
-<?php
-
-static class Foo
-{
-    static function Foo()
```

Comment:
Static constructors are not valid either (PHP 4-7) so the test will stay invalid.
I'll revert for now as to not stop this PR from going forward.